### PR TITLE
Print err msgs if rz_debug_trace_ins_before() fails

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This pr makes `rz_debug_trace_ins_before()` print the reason if it fails. The code is based on a commit in #3066.

**Test plan**

The error messages are ok. All builds are green.

**Closing issues**

...

### CODE REVIEW

File: librz/debug/trace.c [hunk 1]
```diff
@@ -51,16 +51,20 @@ RZ_API bool rz_debug_trace_ins_before(RzDebug *dbg) {
 	// Analyze current instruction
 	ut64 pc = rz_debug_reg_get(dbg, dbg->reg->name[RZ_REG_NAME_PC]);
 	if (!dbg->iob.read_at) {
+		eprintf("Error: dbg->iob.read_at missing\n");
```
Comment (wargio) [inline librz/debug/trace.c]: use RZ_LOG_ERROR

Comment (kazarmy) [reply librz/debug/trace.c]: ok 6328a89f2eb37f5b0eabb1977598ce097088f010

File: librz/debug/trace.c [hunk 2]
```diff
@@ -51,20 +51,20 @@ RZ_API bool rz_debug_trace_ins_before(RzDebug *dbg) {
 	// Analyze current instruction
 	ut64 pc = rz_debug_reg_get(dbg, dbg->reg->name[RZ_REG_NAME_PC]);
 	if (!dbg->iob.read_at) {
-		eprintf("Error: dbg->iob.read_at missing\n");
+		RZ_LOG_ERROR("Error: dbg->iob.read_at missing\n");
```
Comment (wargio) [inline librz/debug/trace.c]: This will output `Error: Error:  dbg->iob.read_at missing`

Comment (wargio) [reply librz/debug/trace.c]: same for the others below

Comment (kazarmy) [reply librz/debug/trace.c]: oh sorry c89b4c49914a50ed4fad91124cc9a9675c9a1b5a

**Review by wargio — APPROVED** [review]

**Pull request merged**